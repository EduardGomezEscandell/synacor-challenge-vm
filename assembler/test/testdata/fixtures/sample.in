push r0 'a'
loop:
    add r0 r0 1
    eq r0 'z'
    out r0
    jt loop
