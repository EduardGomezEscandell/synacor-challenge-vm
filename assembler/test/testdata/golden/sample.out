<VERB push> <REGISTER 0> <CHARACTER a> <EOL>
<TAG_DECL loop> <EOL>
<VERB add> <REGISTER 0> <REGISTER 0> <NUMBER 1> <EOL>
<VERB eq> <REGISTER 0> <CHARACTER z> <EOL>
<VERB out> <REGISTER 0> <EOL>
<VERB jt> <TAG_REF loop> <EOL>
<END>
